R/derive_joined.R

```diff
@@ -81,16 +93,29 @@
 #'
 #'   *Permitted Values*: list of variables or named expressions created by `exprs()`
 #'
-#' @param filter_add Filter for additional dataset (`dataset_add`)
+#' @param first_cond_lower Condition for selecting range of data (before)
 #'
-#'   Only observations from `dataset_add` fulfilling the specified condition are
-#'   joined to the input dataset. If the argument is not specified, all
-#'   observations are joined.
+#'   If this argument is specified, the other observations are restricted from
+#'   the first observation before the current observation where the specified
+#'   condition is fulfilled up to the current observation. If the condition is
+#'   not fulfilled for any of the other observations, no observations are
+#'   considered.
 #'
-#'   Variables created by `order` or `new_vars` arguments can be used in the
-#'   condition.
+#'   This parameter should be specified if `filter_join` contains summary
+#'   functions which should not apply to all observations but only from a
+#'   certain observation before the current observation up to the current
+#'   observation. For an example see the last example below.
```

Comment:
Is this paragraph correct?  current obs up to the current obs?  sounds really strange

Reply:
I think it is correct but maybe it could be phrased better. What I want to say is that an observation before the current observation is selected and then all observations from the selected observation up to the current observation are considered. Any suggestions?

Reply:
Ummm...I understand it now.  I tried wordsmithing it, but mine was just as confusing.  Looking at the last example helped as well.  Let's just leave it as is.